pallets/auctions/src/types.rs

```diff
@@ -0,0 +1,243 @@
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+use sp_runtime::RuntimeDebug;
+use ternoa_primitives::marketplace::MarketplaceId;
+
+#[derive(Encode, Decode, Clone, PartialEq, RuntimeDebug, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+/// Structure to store Auction data
+pub struct AuctionData<AccountId, BlockNumber, BalanceCaps>
+where
+    AccountId: Clone + Default,
+    BalanceCaps: Clone + Default,
```

Comment:
`Balance` is sufficient. BalanceCaps would imply that we are using more than one currency which is not the case.

Reply:
Same for other places where you use BalanceCaps